nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/config/Conv1DConfig.java

```diff
@@ -55,7 +55,7 @@ public Conv1DConfig(long k, long s, long p, long d, String dataFormat, @NonNull
         validate();
     }
 
-    public Conv1DConfig(long k, long s, long p, String dataFormat, boolean isSameMode) {
+    public Conv1DConfig(@NonNull long k,@NonNull long s,@NonNull long p,@NonNull String dataFormat, boolean isSameMode) {
```

Comment:
No `@NonNull` on primitive types - `long` in this instance. Primitive types can't be null.
Also in Conv2DConfig, Conv3DConfig, etc